schemas/context/media-timed-asset-view-details.schema.json

```diff
@@ -55,6 +65,11 @@
           "type": "string",
           "description": "The programmer network name."
         },
+        "xdm:radioStation": {
```

Comment:
Isn't there a pre-existing media grammar that could be used?  Do we really need to invent a new one??

Reply:
Changed this with a reference to ID3.